python/tank/authentication/sso_saml2/core/authentication_session_data.py

```diff
@@ -59,7 +60,7 @@ def __repr__(self):
         :returns: A string containing all of the session data.
         """
         params = {}
-        for key, value in vars(self).iteritems():
+        for key, value in six.iteritems(vars(self)):
```

Comment:
Again, I this is a very short list of things... `items()` is not going to be that more expensive to call.

Reply:
✅